contracts/controller/src/prices.rs

```diff
@@ -22,17 +23,27 @@ impl Contract {
         // Update & insert operation
         self.prices.insert(&price.asset_id, &price);
     }
+
+    pub fn get_markets(&self) -> HashMap<AccountId, AccountId> {
+        let mut result: HashMap<AccountId, AccountId> = HashMap::new();
+        for (asset, asset_contract) in self.markets.iter() {
```

Comment:
@dmytro-shash it's considered as bad practice to use hand-written loops.

Please check if it possible to create HashMap from UnorderedMap.iter()
https://doc.rust-lang.org/std/collections/hash_map/struct.HashMap.html#impl-FromIterator%3C(K%2C%20V)%3E

Also HashMap has several ways to create from Iter, from vector etc.